Formula/dracula-theme.rb

```diff
@@ -0,0 +1,26 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class DraculaTheme < EmacsFormula
```

Comment:
For themes, I've been using the naming scheme `<name>-emacs` ([`dunn/emacs/ahungry-emacs.rb`](https://github.com/dunn/homebrew-emacs/blob/master/Formula/ahungry-emacs.rb), [`dunn/emacs/solarized-emacs.rb`](https://github.com/dunn/homebrew-emacs/blob/master/Formula/solarized-emacs.rb)); so could you rename this to `dracula-emacs.rb`?